Automattic/sensei
# Fix font and content styling issues

### Changes proposed in this Pull Request

* Add missing list styles
* Switch heading fonts to Inter
  * Source Serif wasn't looking that good in big sizes, and this brings some connection between the UI and the content
* Load Inter font from Google Fonts too
* Clean up unused border-color from theme.json
* Adjust margins in a few places

### Testing instructions

* Open a lesson in Learning Mode with rich content, various blocks
* Check that everything looks good


### Screenshot / Video

<img width="1053" alt="image" src="https://user-images.githubusercontent.com/176949/151447832-3524aa84-ff70-4320-968b-a993513340d5.png">
